### PR TITLE
Bug fix for bug introduced in #279

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Unfortunately the bug had been introduced trying to simplify the code, and no test was made for that :S.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
Hi @henryiii, have you got any idea on how to solve this MyPy error? I'm lost in translation.

**Comment by henryiii** [comment]
Probably, I can take a look in a hour or two.

File: src/particle/particle/particle.py [hunk 1]
```diff
@@ -427,7 +429,7 @@ def to_list(
 
     @classmethod
     def to_dict(cls, *args, **kwargs):
-        # type: (Any, Any) -> Dict[str, List[Union[str, float, bool, int]]]
+        # type: (Any, Any) -> Dict[str, Tuple[Union[bool, int, str, float], ...]]
```
Comment (henryiii) [inline src/particle/particle/particle.py:431]: This returns a tuple, not a list.

Comment (henryiii) [reply src/particle/particle/particle.py:431]: And the key is a string, it's the value that's a Tuple.

Comment (eduardo-rodrigues) [reply src/particle/particle/particle.py:431]: I had a Tuple before in my many trials but the hints were tricking me. For sure `str, Tuple` makes sense. I was definitely missing the `...`. Thanks a lot!

**Pull request merged**